### DISCUSSION TITLE
Design and Concept.

### DISCUSSION THREAD

**Discussion by GillianPerrett** [delightedbydhamma/impossible-pali-project#7] [General]
This needs analysis is the first thing to be discussed and agreed upon.  It looks simplistic but if we aren't in agreement about the foundations the building will totter!

Thoughts please!

[IPP Needs Analysis.pdf](https://github.com/delightedbydhamma/impossible-pali-project/files/11074059/IPP.Needs.Analysis.pdf)

**Comment by delightedbydhamma** [comment]
Wonderful. Thanks for this. I totally agree with your conclusion and will spend more time reading your research at some point soon.

I am hoping that at some point the app can learn about each user’s preferred learning style so that we can give them the kinds of exercises that work for them. So if we see that the learner does better with synthetic style tasks we can give them the exercises that they learn better with. This means creating exercises which we ‘tag’ with the different learning styles. The first few levels everyone will get the same exercises but we can then use the data from their results to customise the experience .

**Reply by GillianPerrett** [reply]
That would be beautiful.

**Comment by GillianPerrett** [comment]
Here are some thoughts on a model of language to adopt, for us to bounce off.
If you saw this before, please delete it, I made some changes today following helpful input from Ayya Suvira.

THIS FIRST  VERSION HAS BEEN REVISED IN RESPONSE TO  AYYA'S COMMENTS AND REPLACED WITH VERSION B BELOW:

[2 IPP Language Model.pdf](https://github.com/delightedbydhamma/impossible-pali-project/files/11169225/2.IPP.Language.Model.pdf)

**Reply by delightedbydhamma** [reply]
I got to the bottom and was expecting a section on Meaning, to match your 3 sections of Phonology, lexogrammar and meaning. The section called writing seems like it belongs under phonology. Maybe I missed your point?
Either way I would love to hear more about the Meaning aspect.

Moving on to the nitty-gritty of the phonology I'm not in 100% agreement with the IPA used in this document.
The items in the dental/alveolar column should, in my mind be written as dentals
as in tena:  t̪eːn̪ə  or disvå ˈd̪ɪsˑvɑː
this bring me to our other point of contention which is short a's. 
I’ve been using ə at the ends of words, and ʌ elsewhere. This is my Sri Lankan trained ear, where I was told to make this modification to the end of words. This is something Su, a North Indian speaker, doesn't like. I've forwarded the email to Gillian but don't have permission to share her comment here. Basically her argument is that it changes the meter. This was something that was bothering me as I have been working on my recitation. On this topic, we have to be aware of the modern morph in both north and south indian languages, and Thai to not even pronounce the a on the ends of words. This is the reason I can't just feed the text-to-speech devenagari or sinahala characters and get the results I would want. t̪ʌsːsə becomes t̪ʌsːs

**Reply by GillianPerrett** [reply]
GENERAL:

**I was expecting a section on Meaning**,  .... Oooops!

**The section called writing seems like it belongs under phonology. Maybe I missed your point?** --  My point  (which was missable because I didn't actually state it) is that writing systems are actually secondary to natural languages, which developed with phonology or signs and didn't need tools. ... But actually I guess there are machine languages which have only writing systems.  I should revise my view  along the lines of language not being language if it doesn't have even one a physical system to express it.  Alphabets and syllabries are secondary in that they developed as attempts to represent the sounds.  (Compare Chinese characters which developed to represent meanings.) Point = writing is secondary to phonology intrinsically, and the history of Pali makes this clear: the EBTs were an oral tradition for 200 years.

**Reply by delightedbydhamma** [reply]
I was understanding _meaning_ in the sense of comprehension. As in (most) people who make 'Pali parrot noises' (PPN? 🤨), rather than knowing the meaning. I'm really hopeful that having this auditory resource will help in that sense. At least the parrot noises are mapped to phrases.

**Reply by GillianPerrett** [reply]
That's what's difficult about meaning .... comprehending it ... lol ....  im working on it. .... Tho this could be a good moment to ask Chat GPT4 what it thinks.  ;)

**Comment by GillianPerrett** [comment]
PHONOLOGY:

I have no attachment to one IPA interpretation  over the other.  I would consider minor differences to be due to dialect.  I confess to not having worked with IPA since I was a student c1983.

I screenshotted this version of the chart (rather than some other) for 3 reasons that don't relate to IPA:
1. it was to hand.
2. the rest of the Wikipedia page is easily accessible to anyone who wants to read up on the details.
3. Sujato also used it in class that I link to, so I thought it must be a good one.
Have you listened to his class?  Can you resolve the matter by discussing it with him?   I'm no expert so I tend to lean towards Sujato + Wikipedia unless there is a good reason for leaning otherwards.  Dental/alveolar is  only a tiny bit different from dental.    (It could be another Sri Lankan thing. Maybe?)
 
**I’ve been using ə at the ends of words, and ʌ elsewhere.**. That is definitely a Sri Lankan thing.  Sujato discussed this as well.  I don't recall if he said anything about metre, but it sounds like a persuasive point.

**Reply by delightedbydhamma** [reply]
After listening to Polymothy's vowels IPA lesson I suspect that the short-a is one of those sounds not so common to our ear https://youtu.be/tpR5GdKXpGc?t=496
I also consulted IPA charts for Sinhala, Hindi/urdu and Punjabi. Hindi and Punjabi are the same so I've included them on the same line. Basically, my ear has been corrupted by Sri Lankan friends! 😭 
|                          |short-a      |long-a          |
|---------------|-----------|------------|
| Sinhala             |  ʌ               | ɑː                 |
| Hindi  & Punjabi | ə              | aː                 |
| Pali (wikipedia) | ɐ                 | aː               |


I also found this note about dentals on the Hindi IPA
> Hindi and Urdu contrast dental [[t](https://en.wikipedia.org/wiki/Voiceless_dental_plosive)] and [[d](https://en.wikipedia.org/wiki/Voiced_dental_plosive)] with apical postalveolar [[ʈ](https://en.wikipedia.org/wiki/Voiceless_retroflex_plosive)] and [[ɖ](https://en.wikipedia.org/wiki/Voiced_retroflex_plosive)] (as well as aspirated variants). Both sets sound like [/t/](https://en.wikipedia.org/wiki/Help:IPA/English) and [/d/](https://en.wikipedia.org/wiki/Help:IPA/English) to most English speakers.

I watched Bhante Sujato's talk on the pronunciation and saw how we, as aspirating consonant speakers, over exaggerate our aspirants to make our non-aspirants seem different :sigh:.

**Reply by GillianPerrett** [reply]
I went through hell trying to get the various  t/d possibilities learning Bengali.
You are being very very thorough: which is good because you are the one in our group best able to sort out Pali sounds. 
Sadhu x3.

**Comment by GillianPerrett** [comment]
Sorry it has taken so long to address the matter of meaning and clarify my ideas on writing.  Version B is attached.

You will be wishing that I had been more explicit about chunking, I know Ayya.  That is COMING SOON, currently in the form of a very poor draft.

[2 IPP Language Model B.pdf](https://github.com/delightedbydhamma/impossible-pali-project/files/11169319/2.IPP.Language.Model.B.pdf)

**Comment by GillianPerrett** [comment]
At last:  my thoughts on chunking.  I found sorting them out very fruitful and that they propelled me towards some coherent ideas on syllabus/app design.

EDIT: I am removing the first version of this paper, due to it having certain demerits.
Version B coming shortly.

**Comment by GillianPerrett** [comment]
Here is the revised version.  I realised that I was muddling two different types of chunking in my own mind.
I also realised that I had failed to convey to Ayya my ideas about the presentation of chunks to students.  I'm holding these back until I can stop saying "like App X" or "not like App Y" and just show them.

[3 IPP On Chunking B.pdf](https://github.com/delightedbydhamma/impossible-pali-project/files/11351216/3.IPP.On.Chunking.B.pdf)

**Comment by GillianPerrett** [comment]
When I was at Santi ten days ago, Ayya @delightedbydhamma and I felt that we're  ready to  go ahead simultaneously with **some first tasks**:
1. Developing sound capability using recording from Ayya Achala. (Ayya, @michaelh-sc)
2. Imagining the interface  (Ayya)
3. Character design (Ayya)
4. Collecting ideas for learning activities (everybody: see Discussion Thread #3 and add more)
5. Selecting learning 'mini texts' from MN. (Gillian)

**Additional things coming soon or later**:
1. Inviting others to join the project and their likely roles
2. How to collect stats on learner progress 
3. What our incentive/gaming scheme should look like
4. ??
_This post is editable, so please keep adding/adjusting._
